sphinx/writers/texinfo.py

```diff
@@ -312,15 +312,15 @@ def collect_node_menus(self) -> None:
             del node_menus[top['node_name']]
             top['node_name'] = 'Top'
         # handle the indices
-        for name, content in self.indices:
+        for name, _content in self.indices:
             node_menus[name] = []
             node_menus['Top'].append(name)
 
     def collect_rellinks(self) -> None:
         """Collect the relative links (next, previous, up) for each "node"."""
         rellinks = self.rellinks
         node_menus = self.node_menus
-        for id, entries in node_menus.items():
+        for id in node_menus.keys():
```

Comment:
Is `keys()` call needed?
```suggestion
        for id in node_menus:
```

Reply:
No, not required. Will update